src/drawer.js

```diff
@@ -383,6 +383,7 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
 
         this.context.save();
         this.context.globalAlpha = opacity;
+        this.context.globalCompositeOperation = compositeOperation;
```

Comment:
compositeOperation is optional, so we should only set it to the context if it exists. 

Also, please add compositeOperation to the doc comment here.